ui-new/html/deposit-network-down.html

```diff
@@ -0,0 +1,34 @@
+<section class="viewport deposit_network_down_state system-color" data-tr-section="deposit-network-down">
+  <section class="mainv">
+    <section class="header">
```

Comment:
There is a html `<header>` this is not that important and I'm also using `<section class="main"> ` instead of `<main>` right now.